src/main/scala/com/aergonaut/lifeaquatic/item/armor/SwimTrunkFaceMask.scala

```diff
@@ -0,0 +1,22 @@
+package com.aergonaut.lifeaquatic.item.armor
+
+import com.aergonaut.lifeaquatic.constants.Names
+import net.minecraft.enchantment.Enchantment
+import net.minecraft.entity.player.EntityPlayer
+import net.minecraft.item.ItemStack
+import net.minecraft.potion.{Potion, PotionEffect}
+import net.minecraft.world.World
+
+class SwimTrunkFaceMask extends SwimTrunkArmor(Names.Items.Armor.SwimTrunkFaceMask, 0) {
+  override def onArmorTick(world: World, player: EntityPlayer, itemStack: ItemStack): Unit = {
+    if (player.isInWater()) {
+        player.addPotionEffect(new PotionEffect(Potion.nightVision.id, 10,1))
+    }
+  }
+
+  override def onCreated(itemStack: ItemStack, world: World, player: EntityPlayer)
+  {
+    itemStack.addEnchantment(Enchantment.aquaAffinity,3);
+  }
+
+}
```

Comment:
File also needs a newline at the end. You can setup IntelliJ to do it for you in Preferences > Editor > General > Other > Ensure line feed at end of file